nuxt-pinia-tailwind/stores/counterStore.ts

```diff
@@ -0,0 +1,7 @@
+import { defineStore } from 'pinia'
+
```

Comment:
Should we add an interface for the state, so we can implement it in the useCounterStore?